### PR TITLE
RCPS-315 Create Backend for View Functionality

### PR DESCRIPTION
**Pull request by daiahnadine**

### CODE REVIEW

File: src/main/java/com/rocs/osd/controller/csSlip/CsSlipController.java [hunk 1]
```diff
@@ -1,7 +1,6 @@
 package com.rocs.osd.controller.csSlip;
 
 import com.rocs.osd.domain.csSlip.CsSlip;
-import com.rocs.osd.domain.violation.Violation;
```
Comment (RoCS2024) [inline src/main/java/com/rocs/osd/controller/csSlip/CsSlipController.java:4]: why did you remove this?

File: src/main/java/com/rocs/osd/controller/person/PersonController.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import com.rocs.osd.service.person.PersonService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```
Comment (RoCS2024) [inline src/main/java/com/rocs/osd/controller/person/PersonController.java]: why do you need person controller when you have controllers for each inherited class of Person?

File: src/main/java/com/rocs/osd/service/person/PersonService.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.rocs.osd.service.person;
+
+import com.rocs.osd.domain.person.Person;
+
```
Comment (RoCS2024) [inline src/main/java/com/rocs/osd/service/person/PersonService.java]: Why create a Person service?

File: src/main/java/com/rocs/osd/service/person/impl/PersonServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import com.rocs.osd.domain.person.Person;
+import com.rocs.osd.repository.person.PersonRepository;
+import com.rocs.osd.service.person.PersonService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PersonServiceImpl implements PersonService {
```
Comment (RoCS2024) [inline src/main/java/com/rocs/osd/service/person/impl/PersonServiceImpl.java]: Why create a Person service?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

File: src/main/java/com/rocs/osd/controller/csSlip/CsSlipController.java (hunk 1 above)
Comment (daiahnadine) [reply src/main/java/com/rocs/osd/controller/csSlip/CsSlipController.java:4]: I just noticed that the import does not have any usage po, Sir.

**Review by RoCS2024 — APPROVED** [review]
I missed the PersonRepository in the first review, kindly remove in the next commit. approved for now
